src/scrapers/bgt_scraper.py

```diff
@@ -0,0 +1,91 @@
+"""
```

Comment:
Als naam misschien beter `ams_scraper.py`, dan is het duidelijk dat het alleen om Amsterdam gaat. En dan in de docstring `This script scrapes building footprint data from the Amsterdam BGT API.`